flask_restful/reqparse.py

```diff
@@ -187,19 +190,19 @@ def parse(self, request, bundle_errors=False):
 
                     try:
                         value = self.convert(value, operator)
-                    except Exception as error:
+                    except ValueError as error:
                         if self.ignore:
                             continue
                         return self.handle_validation_error(error, bundle_errors)
+                    except Exception:
+                        if self.ignore:
```

Comment:
Is this really considered a type-conversion failure?  I understood the point of this patch to be that other exceptions are considered programming errors, so this entire ``except`` clause could be dropped.